src/Stratis.Bitcoin/Consensus/Rules/ConsensusRule.cs

```diff
@@ -85,8 +85,13 @@ public abstract class RuleAttribute : Attribute
     /// </remarks>
     public class PartialValidationRuleAttribute : RuleAttribute
     {
+        public PartialValidationRuleAttribute()
+        {
+            this.CanSkipValidation = true;
+        }
+
         /// <summary>A flag that specifies the rule can be skipped when the <see cref="RuleContext.SkipValidation"/> is set.</summary>
-        public bool CanSkipValidation { get; set; } = true;
+        public bool CanSkipValidation { get; set; }
```

Comment:
Now how about the rule number 14? We always put methods below fields and properties.

Reply:
Actually there are no fields or properties

Reply:
not any more (used to be before last commit), but still - https://github.com/dangershony/StratisBitcoinFullNode/blob/97b0d96d57dd59d582b6692e289d8e33e26a95b1/src/Stratis.Bitcoin/Consensus/Rules/ConsensusRule.cs - look at ConsensusRuleDescriptor